src/calculateRentalCost.js

```diff
@@ -28,7 +28,17 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  let total = 0;
+
+  if (days >= 3 && days <= 6) {
+    total = days * 40 - 20;
+  } else if (days >= 7) {
+    total = days * 40 - 50;
+  } else {
+    return days * 40;
+  }
```

Comment:
It's a good practice to avoid `else if` nesting. 
You can apply early return
Example
```
if (condition1) {
 return 1
}

if (condition2) {
 return 2
}

return 3
```